lib/views/pages/post/post_page_widget/bottom_bar_add_comment.dart

```diff
@@ -0,0 +1,56 @@
+import "package:flutter/material.dart";
+
+class BottomBarAddComment extends StatelessWidget {
+  static const commentUserAvatarKey = Key("commentUserAvatar");
+  static const addCommentTextFieldKey = Key("addCommentTextField");
+  static const postCommentButtonKey = Key("postCommentButton");
+
+  final String currentDisplayName;
+
+  const BottomBarAddComment({
+    super.key,
+    required this.currentDisplayName,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment
+          .start, // Align items to the start of the cross axis
+      children: [
+        Padding(
+          padding: const EdgeInsets.only(right: 8),
+          child: CircleAvatar(
+            key: commentUserAvatarKey,
+            radius: 22,
+            child: Text(currentDisplayName.substring(0, 1)),
+          ),
```

Comment:
Similar code already exists in `user_bar_widget`, we should put it in a util `user_avatar` widget and reuse it. I understand that avatars will change in the future, but that is a good reason to refactor and use the same widget everywhere.

Reply:
Following a discussion with the team we though that this should be a separated task which fixes more globally: #157.

I created this new component in a separated PR: #158 based on this current PR.